pallets/msa/src/lib.rs

```diff
@@ -562,37 +562,43 @@ pub mod pallet {
 		#[pallet::weight(T::WeightInfo::add_public_key_to_msa())]
 		pub fn add_public_key_to_msa(
 			origin: OriginFor<T>,
-			msa_owner_public_key: T::AccountId,
+			delegator_key: T::AccountId,
```

Comment:
I don't think we want to confuse adding keys with delegating. I would suggest keeping the original name.